scripts/build/builders/nrf.py

```diff
@@ -161,7 +161,7 @@ def generate(self):
                 zephyr_base = os.environ['ZEPHYR_BASE']
                 nrfconnect_sdk = os.path.dirname(zephyr_base)
                 zephyr_sdk_dir = os.environ.get('ZEPHYR_SDK_INSTALL_DIR') or os.path.join(
-                    os.environ['NRF5_TOOLS_ROOT'], 'zephyr-sdk-0.15.2')
+                    os.environ['NRF5_TOOLS_ROOT'], 'zephyr-sdk-0.16.0')
```

Comment:
in hindsight this was NOT safe: this makes things break for all builders that have not yet upgraded to the new dockerfile.

Reply:
change reverted in #27015